components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -976,8 +993,13 @@ private boolean isUsernameCaseSensitive(String tenantQualifiedUsername) {
     @Deprecated
```

Comment:
mention both getOIDCIdTokenAudiences and getOIDCAccessTokenAudiences as the latter is used during JWT generation

Reply:
is this addressed?